tests_tf/test_attacks.py

```diff
@@ -40,7 +39,7 @@ def test_generate_np(self):
         perturbation_norm = np.sqrt(np.sum(perturbation**2, axis=1))
         # test perturbation norm
         self.assertTrue(np.allclose(perturbation_norm, self.attack.eps))
-
+        
```

Comment:
Nevermind about creating separate files, as long as the tests are placed in the corresponding test file (i.e., `test_attacks.py` or `test_attacks_tf.py` (which does not exist yet)